tools/vendored.py

```diff
@@ -64,6 +64,17 @@ def rewrite_importlib_resources(pkg_files, new_root):
         file.write_text(text)
 
 
+def rewrite_more_itertools(pkg_files: Path):
+    """
+    Rewrite more_itertools to remove unused more_itertools.more
+    """
+    for more_file in pkg_files.glob("more.py*"):
```

Comment:
Instead of glob, just use `pkg_files.joinpath('more.py')` and apply the changes to that.

Reply:
The intent was to remove also associated `*.pyi` files to prevent typechecker (and possibly also smart editors) from believing the removed content is still there